src/components/organisms/ListDoctor/ListDoctor.tsx

```diff
@@ -15,39 +12,29 @@ interface DataDr {
   address: string;
 }
 
-export const ListDoctor = (props: Props) => {
+export const ListDoctor = () => {
   const [list, setList] = useState([]);
-  const [on, setOn] = useState(false);
+  const [idPt, setIdPt] = useState('');
 
-  const listAll = async (e: SyntheticEvent) => {
-    e.preventDefault();
+  useEffect(() => {
+    getDoctors();
+  });
 
+  useEffect(() => {
+    sendToken(getToken, baseUrlPatient, 'find-doctor').then((r) => setIdPt(r.idPt));
```

Comment:
oby przy każdym requeście nie trzeba było najpierw sprawdzać tokenu osobnym requestem - na backend się token wysyła w nagłówku normalnego requestu. A żeby łatwo się ten token dołączało to możesz skorzystać z `axios` i jego request interceptora.